.github/workflows/build.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Build IDF bootloader and partition table
-        uses: docker://docker.io/espressif/idf:v4.4.1
+        uses: docker://docker.io/espressif/idf:release-v4.4
```

Comment:
What's the difference between the `release-vxx` and `vxxx` tags?

Reply:
The `release-vxx` refers to a release branch, so the image gets updated more frequently.
The `vxxx`, in turn, points to a release tag from IDF.